cni/network/network.go

```diff
@@ -244,7 +244,7 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 		enableInfraVnet  bool
 		enableSnatForDns bool
 		nwDNSInfo        network.DNSInfo
-		cniMetric        telemetry.CNIReport
+		cniMetric        telemetry.AIMetric
```

Comment:
why was this changed?

Reply:
i dont want to change existing CNIReport structure as this schema used by hostnetagent